flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "only-arches": ["x86_64"],
+    "skip-appstream-check": true
```

Comment:
```suggestion
```

Reply:
I think I added it because the current appstream file in Frescobaldi tarball contains an error which doesn't pass validation. That's why there's a patch file for current release. Don't know if the CI script applies the patch before validating...

Anyway, I removed it and let's see what happens...

Reply:
validation by flathub is done at the end, after the package is built. Usually we add the patches to validate instead of skipping the check.

Reply:
As the [latest build](https://flathub.org/builds/#/builders/12/builds/3443) shows, I was right: validation is done before the patches are applied. I will revert the change and put pack skip-appstream-check, Ok?

Reply:
no it is done at the end. `skip-appstream-check` shall only done for unfixable exceptions (when it's not relevant like building extension)

Reply:
The validation fails because there is a URL in the `<description>` and the appstream validation doesn't like that. You could move it to a `<url>` element just outside of `<description>` and it would be fine.